backend/src/hmc-api/fetcher/utils.ts

```diff
@@ -0,0 +1,88 @@
+import * as APIv4 from "hyperschedule-shared/api/v4";
+import { DATA_DIR_PATH } from "hyperschedule-data";
+import type { Endpoint, Endpoints } from "./types";
+import { Params } from "./types";
+import { resolve, dirname } from "path";
```

Comment:
mildly prefer common/general-purpose modules to be imported as `import * as path` to keep global namespace tidier (also below, `import * as fs`)

Reply:
I think everything from `path` should be imported by value because we want to free up the `path` variable name.